commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/bzip2/Bzip2FileSystem.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.commons.vfs2.provider.compressed.CompressedFileFileSystem;
 
 /**
- * Filesytem to handle compressed files using the bzip2 method.
+ * Filesystem to handle compressed files using the bzip2 method.
```

Comment:
This is not right, it should either be "file system" or "FileSystem".

Reply:
Done.